helios-services/src/main/java/com/spotify/helios/master/ZooKeeperMasterModel.java

```diff
@@ -401,28 +416,12 @@ public void removeDeploymentGroup(final String name) throws DeploymentGroupDoesN
     try {
       client.ensurePath(Paths.configDeploymentGroups());
       client.ensurePath(Paths.statusDeploymentGroups());
-      client.ensurePath(Paths.statusDeploymentGroupTasks());
 
       final List<ZooKeeperOperation> operations = Lists.newArrayList();
 
-      // /status/deployment-group-tasks/[group-name] might exist (if a rolling-update is in
-      // progress). To avoid inconsistent state make sure it's deleted if it does exist:
-      //
-      // * If it exists: delete it.
-      // * If it doesn't exist, add and delete it in the same transaction. This is a round-about
-      //   way of ensuring that it wasn't created when we commit the transaction.
-      //
-      // Having /status/deployment-group-tasks/[group-name] for removed groups around will cause
-      // DGs to become slower and spam logs with errors so we want to avoid it.
-      if (client.exists(Paths.statusDeploymentGroupTasks(name)) != null) {
-        operations.add(delete(Paths.statusDeploymentGroupTasks(name)));
-      } else {
-        operations.add(create(Paths.statusDeploymentGroupTasks(name)));
-        operations.add(delete(Paths.statusDeploymentGroupTasks(name)));
-      }
-
       operations.add(delete(Paths.configDeploymentGroup(name)));
       operations.add(delete(Paths.statusDeploymentGroupHosts(name)));
+      operations.add(delete(Paths.statusDeploymentGroupRollingOps(name)));
```

Comment:
Is it intentional that you don't remove `/status/rolling-operations/<uuid>` and `/status/rolling-operations-tasks/<uuid>`?